utils/query_helpers.py

```diff
@@ -0,0 +1,30 @@
+from typing import Any
```

Comment:
Running `isort` here will introduce newlines delimiting built-in, third party, and local code, following [PEP-8](https://peps.python.org/pep-0008/#imports):
```python
from typing import Any

from langchain.callbacks import get_openai_callback

from yolopandas import pd
```

Reply:
Interestingly, I ran isort myself and the newlines were not auto-added for me so I manually added them in.